.drone.yml

```diff
@@ -1,6 +1,13 @@
 workspace:
+  base: /srv/app
   path: src
 
+clone:
+  git:
+    image: plugins/git:latest
+    depth: 50
```

Comment:
Hum, I can also do depth 1... I just copied that part from go-gitea/gitea since it had been there like that as well.